THIRD-PARTY-LICENSES

```diff
@@ -0,0 +1,23 @@
+** g-plane/simple-base -- https://github.com/g-plane/simple-base
```

Comment:
are we on the hook to include this in the npm package or can it stay in the code base? If the requirement is to have it in the npm package, can you check in node_modules if simple-base already has it in there? In that case we may not need to include it.

Reply:
I need a base36 conversion, I cannot find any in `node_modules`. The simple-base is MIT license and we are giving attribution in the THIRD-PARTY-LICENSES which is what is required.

Reply:
If the license is the simple-base npm package already (which it is), we don't need to include it in our code base.
You can check under node_modules/simple-base/LICENSE. Our software won't run without the npm and hence the license will always be there. 

Otherwise you will see a lot more licenses in our code.